### DISCUSSION TITLE
Cannot find configuration files for rootless environment

### DISCUSSION THREAD

**Discussion by AreRex14** [containers/podman#15110] [Q&A]
I am using **Podman 3.4.4** in **Ubuntu 22.04 WSL**. I cannot find the directories and files for the rootless environment. The configuration files are existed for /usr/share/containers and /etc/containers though. Any though? Thanks

Based on below latest tutorial documentation
In the rootless environment they reside in ${XDG_CONFIG_HOME}/containers (usually ~/.config/containers) and are owned by each individual user.

The three main configuration files are [containers.conf](https://github.com/containers/common/blob/main/docs/containers.conf.5.md), [storage.conf](https://github.com/containers/storage/blob/main/docs/containers-storage.conf.5.md) and [registries.conf](https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md). The user can modify these files as they wish.

**Comment by eriksjolund** [comment]
I tried out the web page [Ubuntu Packages Search](https://packages.ubuntu.com) where it is possible to search for files.

For the field
_packages that contain files whose names end with the keyword_
I entered _containers.conf_

For the field _distribution_
I chose _jammy_.  

(Ubuntu 22.04 has also the name _Jammy Jellyfish_)



Result:

https://packages.ubuntu.com/search?searchon=contents&keywords=containers.conf&mode=&suite=jammy&arch=any

File |	Packages
---  |   --------
/usr/share/containers/containers.conf |	golang-github-containers-common
/usr/share/games/opencity/graphism/industrial/containers/containers.conf	| opencity-data
/usr/share/gocode/src/github.com/containers/common/pkg/config/containers.conf |	golang-github-containers-common-dev

Maybe you need to install _golang-github-containers-common_ ?
